src/software/jetson_nano/services/motor.cpp

```diff
@@ -267,11 +275,23 @@ bool MotorService::checkDriverFault(uint8_t motor)
 TbotsProto::MotorStatus MotorService::poll(const TbotsProto::MotorControl& motor,
                                            double time_elapsed_since_last_poll_s)
 {
+    TbotsProto::MotorStatus motor_status;
+
+    bool encoders_calibrated = (encoder_calibrated_[FRONT_LEFT_MOTOR_CHIP_SELECT] ||
+                                encoder_calibrated_[FRONT_RIGHT_MOTOR_CHIP_SELECT] ||
+                                encoder_calibrated_[BACK_LEFT_MOTOR_CHIP_SELECT] ||
+                                encoder_calibrated_[BACK_RIGHT_MOTOR_CHIP_SELECT]);
+
+    int reset_detector = tmc4671_readInt(0, TMC4671_PID_ACCELERATION_LIMIT);
+
+    if (reset_detector == 2147483647)
```

Comment:
nit: Can we refactor the constant here (`2147483647`), maybe to the header file or to `constants.h` and add a comment describing the value a bit more and maybe how we know that this is the value (i.e. was it taken from some documentation that we can link)?